PopupMenu/PopupMenu.js

```diff
@@ -53,7 +59,7 @@ const PopupMenuBase = kind({
 					<Cell className={css.title} shrink>
 						<Heading size="title">{title}</Heading>
 					</Cell>
-					<Cell className={css.body} shrink>
```

Comment:
Was the `shrink` prop not cooperating with the new `Scroller`?

Reply:
`shrink` was originally removed because it was not cooperating with Scroller's auto-sizing. Moving Scroller into the Cell as a child allowed to Scroller to set itself to the correct dimensions, and then Cell could shrink-wrap around Scroller so they can both be positioned automatially by flex-box.